vital/algo/pointcloud_io.cxx

```diff
@@ -0,0 +1,69 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/// \file
+/// \brief Implementation of save wrapping functionality
+
+#include "pointcloud_io.h"
+
+#include <vital/algo/algorithm.txx>
+#include <vital/exceptions/io.h>
+#include <kwiversys/SystemTools.hxx>
+
+/// \cond DoxygenSuppress
+INSTANTIATE_ALGORITHM_DEF( kwiver::vital::algo::pointcloud_io );
+/// \endcond
+
+namespace kwiver {
+namespace vital {
+namespace algo {
+
+pointcloud_io
+::pointcloud_io()
+{
+  attach_logger( "algo.pointcloud_io" );
+}
+
+void
+pointcloud_io::
+save(vital::path_t const& filename,
```

Comment:
The original standalone function had two signatures and I've copied them into this interface as well. I felt like this might need some refactoring so I'd appreciate feedback on whether to remove it.
The first signature here isn't used in any KWIVER applets and is solely used on [this line in TeleSculptor](https://github.com/Kitware/TeleSculptor/blob/master/gui/MainWindow.cxx#L2398).

Reply:
Answered my own question, I removed the signature used in TeleSculptor. It can be explicitly implemented in TeleSculptor where it needs to be added.

Reply:
was that the signature for saving landmarks?  If so, I would keep it in here.